constants/progresses.ts

```
@@ -22,6 +22,8 @@ const PROGRESSES_SIZE = 20;
 const PROGRESSES_PAGE_SIZE = 0;
 const VALID_PROGRESS_TYPES = ["task", "user"];
 
+const UNAUTHORIZED_WRITE = "Unauthorized to write progress of task"
```

ESLint

[error] Insert `;`

(prettier/prettier)